packages/shared-ux/page/solution_nav/src/with_solution_nav.tsx

```diff
@@ -53,11 +51,11 @@ export const withSolutionNav = <P extends TemplateProps>(WrappedComponent: Compo
       isMediumBreakpoint || (canBeCollapsed && isLargerBreakpoint && !isSideNavOpenOnDesktop);
     const withSolutionNavStyles = WithSolutionNavStyles(euiTheme);
     const sideBarClasses = classNames(
-      'kbnStickyMenu',
```

Comment:
This is no longer needed as `sticky` is now set through `page_template_inner`: https://github.com/elastic/kibana/blob/main/packages/shared-ux/page/kibana_template/impl/src/page_template_inner.tsx#L74